charts/ccd-definition-store-api/values.preview.template.yaml

```diff
@@ -23,9 +23,11 @@ java:
     # enable whenever required and provide host url to match with corresponding data-store-api
     ELASTIC_SEARCH_ENABLED: true
 
-    ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-1260-es-master
+    # TODO: remove TEMPORARY OVERRIDE to test ElasticSearch, reinstate: ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-1260-es-master
+    ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-2086-es-master
```

Comment:
This should be reveresed if final test-def version changes are merged to `master`.